src/sentry/conf/server.py

```diff
@@ -1883,6 +1883,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:slack-thread": False,
     # Send Slack notifications to threads for Issue Alerts
     "organizations:slack-thread-issue-alert": False,
+    # Use SNQL table join
+    "organizations:snql-join": False,
```

Comment:
will this feature be used only in this part of the codebase? is there a more descriptive comment that could be used?

Reply:
I left it vague since we have at least one more feature that wants to implement this, and to be safe it's being rolled out slowly. There are Datadog dashboards for this that can monitor the load. I could make it more specific to this area and then the other feature can use another flag if need be.